sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ArithmeticExpressionSuite.scala

```diff
@@ -142,4 +142,22 @@ class ArithmeticExpressionSuite extends SparkFunSuite with ExpressionEvalHelper
     checkEvaluation(MinOf(Literal.create(null, IntegerType), 1), 1)
     checkEvaluation(MinOf(1, Literal.create(null, IntegerType)), 1)
   }
+
+  test("pmod") {
+    checkEvaluation(Pmod(7, 3), 1)
```

Comment:
can we use testNumericDataTypes, and then add extra test cases for floating points?